crates/masonry/src/widget/button.rs

```diff
@@ -82,13 +82,18 @@ impl Widget for Button {
                 }
             }
             PointerEvent::PointerUp(_, _) => {
-                if ctx.is_active() && !ctx.is_disabled() {
+                if ctx.is_active() && ctx.is_hot() && !ctx.is_disabled() {
```

Comment:
Just curious when this would be active, but not hot?

Reply:
When you have started holding down the mouse, moved it away, then raised the mouse.